java/src/org/dartlang/vm/service/RemoteServiceRunner.java

```diff
@@ -21,9 +21,8 @@
 public interface RemoteServiceRunner {
```

Comment:
You could also have `RemoteServiceRunner` have a method like `getServiceName()`, so you don't need to pass the serviceName into the addServiceRunner method - a RemoteServiceRunner would know both its name and how to handle service call.s